spec/client_spec.rb

```diff
@@ -5,22 +5,33 @@
 
 describe 'Ephemeral::Client' do
   
-  it 'expects 3 arguments' do
-    test_client = Ephemeral::Client.new
-    expect{
-      test_client.build
-    }.to raise_error ArgumentError
-  end
+  describe 'build' do
+    it 'expects 3 arguments' do
+      test_client = Ephemeral::Client.new
+      expect{
+        test_client.build
+      }.to raise_error ArgumentError
+    end
+
+    it 'returns a hash' do
+      test_client = Ephemeral::Client.new
+      response = test_client.build("ruby:2.1", "https://github.com/skierkowski/hello-middleman", "middleman")
+      expect(response).to be_an_instance_of Hash
+    end
 
-  it 'returns a hash' do
-    test_client = Ephemeral::Client.new
-    response = test_client.build("ruby:2.1", "https://github.com/skierkowski/hello-middleman", "middleman")
-    expect(response).to be_an_instance_of Hash
+    it 'returns a hash which contains an id' do
+      test_client = Ephemeral::Client.new
+      response = test_client.build("ruby:2.1", "https://github.com/skierkowski/hello-middleman", "middleman")
+      expect(response).to have_key('id')
+    end
   end
 
-  it 'returns a hash which contains an id' do
-    test_client = Ephemeral::Client.new
-    response = test_client.build("ruby:2.1", "https://github.com/skierkowski/hello-middleman", "middleman")
-    expect(response).to have_key('id')
+  describe 'getLogs' do
+    it 'returns array from id' do
```

Comment:
since this will fail, keep the `describe` and `it` block, just comment out the tests for now.